scripts/entrypoint.sh

```diff
@@ -13,5 +13,13 @@ usermod -g "$USER_GID" consul 2> /dev/null
 chown -R -h "$USER_UID" "$BUNDLE_PATH"
 chgrp -R -h "$USER_GID" "$BUNDLE_PATH"
 
+# Remove a potentially pre-existing server.pid for Rails.
+set -e
```

Comment:
Just one more thing (as Columbo used to say :smile:): Is this line necessary? :thinking: